estela-api/api/mixins.py

```diff
@@ -12,6 +12,30 @@ class APIPageNumberPagination(PageNumberPagination):
     page_size = settings.API_PAGE_SIZE
     max_page_size = settings.API_MAX_PAGE_SIZE
 
+    def get_paginated_response_schema(self, schema):
```

Comment:
@rodp63 I think this is not having any effects on the final endpoint docs. If this is the case, let's just remove this.